content/en/benchmark.md

```diff
@@ -0,0 +1,128 @@
+# NumPy Benchmarking
+
+
+![output-modify-2-new](https://user-images.githubusercontent.com/62256509/127653679-5151e22d-d141-4621-abb9-b3e6dd7988b7.jpg)
```

Comment:
It would be better to commit the figure in this PR. The place to put it is `static/images/content_images/`